ddp-workspace/projects/ddp-pancan/src/app/components/welcome/stay-informed-section/stay-informed-section.component.ts

```diff
@@ -12,19 +14,24 @@ export class StayInformedSectionComponent {
     @Input() title: string;
     @Input() text: string;
     @Input() btnText: string;
+    @Input() isColorectal: boolean;
```

Comment:
We are going to have pretty the same for LMS page.
Should we modify it somehow as not to duplicate inputs ? 
(e.g. have some enum with `colorectal`/`lms` values or smth else)